client/state/data-layer/wpcom/read/following/mine/new/index.js

```diff
@@ -28,21 +31,39 @@ export function requestFollow( { dispatch }, action ) {
 			},
 			onSuccess: action,
 			onFailure: action,
-		} )
+		} ),
+	);
+
+	// build up a notice to show
```

Comment:
This is okay as long as we don't repeat this any more places + the logic doesn't get any more complex.  

If we ever need to grab titles from feedUrl in the future elsewhere in the data-layer or make this more complicated then lets make a selector that goes straight to the title from things like { site, feed, feedUrl }